src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public Figure getRandomFigure() {
+        new Random();
+        int figureIndex = new Random().nextInt();
+        final ColorSupplier colorSupplier = new ColorSupplier();
+        switch (figureIndex) {
+            case 0:
+                return new Circle(colorSupplier.getRandomColor(), getRandomInt());
+            case 1:
+                return new Square(colorSupplier.getRandomColor(), getRandomInt());
+            case 2:
+                return new RightTriangle(colorSupplier.getRandomColor(),
+                        getRandomInt(), getRandomInt());
+            case 3:
+                return new IsoscelesTrapezoid(colorSupplier.getRandomColor(),
+                        getRandomDouble(), getRandomDouble(), getRandomDouble());
+            case 4:
+            default:
+                return new Rectangle(colorSupplier.getRandomColor(),
+                        getRandomDouble(), getRandomDouble());
+        }
+    }
+
+    private int getRandomInt() {
+        return new Random().nextInt(5);
+    }
+
+    private double getRandomDouble() {
+        return new Random().nextDouble();
+    }
+
+    public void getDefaultFigure() {
```

Comment:
this method should  return this:
![Screenshot 2022-06-07 at 07 55 31](https://user-images.githubusercontent.com/60258373/172299077-6a992c22-0f0d-4500-9934-a66cdd77dbb5.png)